.github/workflows/ci-frontend.yml

```diff
@@ -56,21 +52,22 @@ jobs:
             test-chunk-ids: ${{ steps['set-test-chunk-ids'].outputs['test-chunk-ids'] }}
         steps:
             - uses: actions/checkout@v3
-            - uses: actions/cache@v3
-              id: node-modules-cache
+            - name: Install pnpm
+              uses: pnpm/action-setup@v2.2.4
+              with:
+                  version: 7.x.x
+            - name: Set up Node 16
+              uses: actions/setup-node@v3
               with:
-                  path: |
-                      node_modules
-                  key: ${{ runner.os }}-node-modules-${{ hashFiles('**/yarn.lock') }}
-                  restore-keys: |
-                      ${{ runner.os }}-node-modules-
-            - run: yarn install --frozen-lockfile
+                  node-version: 16
+                  cache: pnpm
+            - run: pnpm install --frozen-lockfile
             - id: set-test-chunks
               name: Set Chunks
-              # Looks at the output of 'yarn test --listTests --json'
-              # Take the 3rd line of the output (the first two are yarn non-sense)
+              # Looks at the output of 'pnpm test -- --listTests --json'
+              # Take the 3rd line of the output (the first two are pnpm non-sense)
               # Split the test into 3 parts. To increase the number split change the denominator in `length / 3`
-              run: echo "test-chunks=$(yarn test --listTests --json | sed -n 3p | jq -cM '[_nwise(length / 3 | ceil)]')" >> $GITHUB_OUTPUT
+              run: echo "test-chunks=$(pnpm test -- --listTests --json | sed -n 3p | jq -cM '[_nwise(length / 3 | ceil)]')" >> $GITHUB_OUTPUT
```

Comment:
Updated 👍 

> Allow edits and access to secrets by maintainers

Apparently GitHub doesn't support this if the repo and the fork are both located in organization accounts rather than personal accounts. 🤷 https://github.com/orgs/community/discussions/5634

Reply:
Oh, that's very annoying. Now there are some "Jest encountered an unexpected token" problems in Frontend CI checks, and unfortunately I'm not able to push a fix either.